docs/assets/js/swagger.yml

```diff
@@ -225,10 +225,11 @@ definitions:
   setup:
     type: object
     properties:
-      display_name:
+      username:
+        description: an identifier for the user (e.g. jane.doe)
         type: string
     required:
-      - display_name
+      - username
```

Comment:
this one looks like it is part of another change that was reverted but a copy left in the docs.